lib/ansible/runner.rb

```diff
@@ -408,20 +410,34 @@ def wait_for_listener_start(listener)
         end
       end
 
-      def python_path
-        @python_path ||= [manageiq_venv_path, *ansible_python_paths].compact.join(File::PATH_SEPARATOR)
+      def runner_python_path
+        @runner_python_path ||= [venv_python_path, ansible_python_path].compact.join(File::PATH_SEPARATOR)
+      end
+
+      def runner_path
+        @runner_path ||= [venv_bin_path, ENV["PATH"].presence].compact.join(File::PATH_SEPARATOR)
+      end
+
+      VENV_ROOT = "/var/lib/manageiq/venv".freeze
+
+      def venv_python_path
+        Dir.glob(File.join(VENV_ROOT, "lib/python*/site-packages")).first
+      end
+
+      def venv_bin_path
+        Dir.glob(File.join(VENV_ROOT, "bin")).first
```

Comment:
This is "clever", but it's effectively a way to check the presence of a directory and return nil if not present